modules/integration/src/main/scala/scala/cli/integration/TestInputs.scala

```diff
@@ -39,17 +39,17 @@ object TestInputs {
     val rng  = new SecureRandom
     val d    = base / s"run-${math.abs(rng.nextInt().toLong)}"
     os.makeDir.all(d)
-    Runtime.getRuntime.addShutdownHook(
-      new Thread("scala-cli-its-clean-up-tmp-dir") {
-        setDaemon(true)
-        override def run(): Unit =
-          try os.remove.all(d)
-          catch {
-            case NonFatal(e) =>
-              System.err.println(s"Could not remove $d, ignoring it.")
-          }
-      }
-    )
+    // Runtime.getRuntime.addShutdownHook(
```

Comment:
How we deal with left over tmp dirs now?

Reply:
My mistake, I only meant to comment that out locally.